package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "videojs-contrib-hls",
-  "version": "1.3.3",
+  "version": "1.3.4",
```

Comment:
I did but then I pushed to development and these commits didn't go away. They are the same commit as the latest master so I think it's just cruft.